lib/src/builder.dart

```diff
@@ -128,19 +133,28 @@ class AggregateTestBuilder extends Builder {
         final path =
             p.relative(id.path, from: p.dirname(buildStep.inputId.path));
         imports.writeln("import '$path' as $prefix;");
-        mains.writeln("  $prefix.main();");
+        mains.add("  $prefix.main();");
       }
     }
 
     // Don't generate an empty aggregate test.
     if (imports.isEmpty) return;
+    if (_config.randomizeOrderingSeed != null) {
+      var seed = _config.randomizeOrderingSeed.toLowerCase() == 'random'
+          ? Random().nextInt(4294967295)
+          : int.parse(_config.randomizeOrderingSeed);
+      log.shout('Shuffling test order with `randomize_ordering_seed: $seed`');
```

Comment:
Ok this was my bad, apparently using `shout()` qualifies as a build failure and thus prevents tests from even running :) whoops!

I realized though that we can just add our own newline to the end here and then the info log will stick around:
```suggestion
      log.info('Shuffling test order with `randomize_ordering_seed: $seed`\n');
```